src/registry/index.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 var async = require('async');
-var colors = require('colors');
+var colors = require('colors/safe');
```

Comment:
colors exposes the classic mode, which just overrides global strings by allowing to do `"string".red` or the "safe" mode `colors.red("string")`. We use the safe mode almost everywhere but we missed this before..